contracts/Shortfall/Shortfall.sol

```diff
@@ -44,13 +51,15 @@ contract Shortfall is Ownable2StepUpgradeable, AccessControlledV8, ReentrancyGua
 
     /// @notice Auction metadata
     struct Auction {
+        /// @notice It holds either the starting block number or timestamp
         uint256 startBlock;
```

Comment:
Why don't we rename it to `startkBlockOrTimestamp`? It's an internal change, so it shouldn't have any impact externally and it would be more precise. The same comment would be for `highestBidBlock`

Reply:
Done https://github.com/VenusProtocol/isolated-pools/commit/6b98b095e5e6b3322b6d0c825ba273612e49948a